packages/babel-parser/test/fixtures/experimental/module-attributes/invalid-syntax-with-attributes-multiple-lines/input.js

```diff
@@ -0,0 +1,3 @@
+import "x"
+with
+type: "json"
```

Comment:
This error is thrown because invalid with statement parsing but not invalid with attributes.
I don't think currently we need testing invalid multiple line attributes because only `"type"` is allowed, but it is worthy to test ASI (it is currently valid).
```js
import "x" with
type: "json"
```